products/compute/terraform.yaml

```diff
@@ -234,6 +235,33 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'compareSelfLinkOrResourceName'
       timeoutSec: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  BackendServiceSignedUrlKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    exclude_import: true
+    is_fine_grained_resource: true
```

Comment:
Can this be a property/convenience method based on the existence of `nested_query` rather than its own property?

Reply:
`:exclude_import` is less because it's fine-grained, more because this has two fields (name and key) and one is sensitive, so it doesn't really make sense to allow for import. Otherwise, we'd have to supply an empty key value, and the user would be asked to apply the key value, which would just recreate the key.

Also, I want to use `nested_query` for collections (i.e. instead of having `self_link_query` for list-only resources) which I do think the current validator code doesn't handle correctly but should include still, vs fine-grained which make less sense to include since they can't be tied to a specific CAI resource.